app/policies/user_policy.rb

```diff
@@ -29,6 +29,11 @@ def member_dashboard?
 
 # CRUD
 
+  # Admins can't delete themselves.
+  def destroy?
+    is_admin? && user != record
```

Comment:
the method answers the question: "Can the currently signed in user (user) delete the given user? (record)"

The answer is "Yes: If the user is an admin and the user it is trying to delete (record) is not themselves (user)"